docs/adr/0016-some-case-list-email-present-wrong.md

```diff
@@ -0,0 +1,21 @@
+# 16. Fix some case list email present wrong
+
+Date: 2022-09-13
+
+## Status
+
+- Issue: Fix some case list email present wrong
+
+## Context
+
+- Root cause: When you get list email with filter at first time, then clear this filter,
+list email cached not enough.
+
+
+## Decision
+
+- Always get full list email at first time without filter.
```

Comment:
- Fill the cache with the first page of this mailbox at the first time open this mailbox